config-templates/ibm/ibm-object-storage-plugin/changelog.json

```diff
@@ -3,25 +3,24 @@
         "2.2": {
             "status": "supported",
             "latest-revision": {
-                "revision": "18",
-                "release-date": "02 February 2024",
-                "cves": "CVE-2023-3446, CVE-2023-3817, CVE-2023-5678",
+                "revision": "19",
+                "release-date": "29 February 2024",
+                "cves": "",
```

Comment:
Please retain `"ubi": "",` field. Somehow we missed in last release.

Reply:
As of now base images are not based on UBI , so ubi version will be irrelevant here.